src/components/common/YourCompany/ChangeBusinessType.vue

```diff
@@ -173,8 +173,15 @@
             <strong class="pl-2 gray9--text text-small-text">Change company name</strong>
           </header>
           <article class="pl-8 pt-1 small-text">
-            To change to a {{ GetCorpFullDescription(selectedEntityType) }}, you must change the company
-            name using an approved name request or change it to a numbered company.
+            To change to a {{ GetCorpFullDescription(selectedEntityType) }}, you must change your company name.
+            You can use your incorporation number as the company name or go to
+            <a
+              href="https://dev.names.bcregistry.gov.bc.ca/"
+              class="underline-blue-text"
```

Comment:
Shall we add the url to the .env file?

Reply:
Yes!

Reply:
Does this link open in a new window/tab? Should it?